### PR TITLE
Add JLab SciToken issuers SOFTWARE-4986

### PR DESCRIPTION
**Pull request by jasoncpatton**

### CODE REVIEW

File: virtual-organizations/EIC.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 AppDescription: EIC collaboration.
 CertificateOnly: false
-Community: The EIC collaboration. 
+Community: The EIC collaboration.
```
Comment (brianhlin) [inline virtual-organizations/EIC.yaml:3]:
```suggestion
Community: The EIC collaboration. 
```

Just to make the diff cleaner

**Review by brianhlin — APPROVED** [review]
LGTM, deleted nitpicky suggestion since I'd like to get this in ASAP

**Comment by jasoncpatton** [comment]
> LGTM, deleted nitpicky suggestion since I'd like to get this in ASAP

That trailing space bugged me while I was putting in the credentials... it will just be annoying in the commit now :)

**Comment by brianhlin** [comment]
I appreciate the fixup! Normally, I do ask folks to separate those sort of changes into whitespace-only commits for cleaner diffs. It was small enough here that it's nbd, though.

**Pull request merged**